library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java

```diff
@@ -816,7 +835,23 @@ private void updateMainFabDrawable(boolean animate) {
         } else {
             UiUtils.rotateBackward(mMainFab, animate);
             if (mMainFabClosedDrawable != null) {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N && mMainFabClosedDrawable instanceof AnimatedVectorDrawable) {
+                    updated = true;
+                    mMainFab.setImageDrawable(mMainFabClosedDrawable);
+                    ((AnimatedVectorDrawable) mMainFabClosedDrawable).start();
+                } else if (Build.VERSION.SDK_INT < Build.VERSION_CODES.N && mMainFabClosedDrawable instanceof AnimatedVectorDrawableCompat) {
```

Comment:
@S-H-Y-A I can't check this myself right now: are you 100% positive that on `VERSION_CODES.N` it is not possible to have `AnimatedVectorDrawableCompat`? What I'm asking is if we could have a case where the `VERSION.SDK_INT` is  `Build.VERSION_CODES.N` but `mMainFabClosedDrawable` is an instance of `AnimatedVectorDrawableCompat`.

(probably not, but I just want to be sure)

Reply:
Yes, it works.

Look at this.
![AnimatedVectorDrawableCompat](https://user-images.githubusercontent.com/74596628/111332751-66fd2200-86b5-11eb-9b46-8994bf8e6bd0.png)

Reply:
Thanks :+1: